cookbooks/travis_build_environment/recipes/hhvm.rb

```diff
@@ -3,7 +3,7 @@
   key = "E7D1FA0C"
 else
   hhvm_uri = 'http://dl.hhvm.com/ubuntu'
-  key = 'http://dl.hhvm.com/conf/hhvm.gpg.key'
+  key = 'http://dl2.hhvm.com/conf/hhvm.gpg.key'
```

Comment:
I think the key URL should also be switched to HTTPS?

Reply:
Ooh, definitely! Thank you for the catch.